warehouse/query-core/src/main/java/datawave/query/index/lookup/Intersection.java

```diff
@@ -219,6 +237,18 @@ public Tuple2<String,IndexInfo> next() {
         return ret;
     }
     
+    /**
+     * This is a no-op for the Intersection class. Delegates to {@link #next()}
+     *
+     * @param context
+     *            a context
+     * @return the result of calling {@link #next()}
+     */
+    @Override
+    public Tuple2<String,IndexInfo> next(String context) {
+        return next();
```

Comment:
couldn't context be used to potentially skip forward?

Reply:
I'll take a look doing this.

Reply:
Added in latest commit